### PR TITLE
docs: add English README.md and CONTRIBUTING.md files

### PR DESCRIPTION
**Pull request by cliffordfajardo**
## What is the change?
- added English translation for `README.md` file
- added English translation for `CONTRIBUTING.md` file

## Why are we making this change?
- Help English speaking users understand what this library offers
- Helps browser search engines surface the XFlow library to English speaking population to help drive adoption of this library by more people

## Demo of Changes
- https://github.com/cliffordfajardo/XFlow

## Notes:
I followed the same README strategy as 
- @antv/graphin README : https://github.com/antvis/Graphin

### CODE REVIEW

**Comment by cliffordfajardo** [comment]
@yn-boyu @lxfu1 - 
please feel free to make edit to my github code branch, github now has Codespaces enabled for nearly all repos on Github.com 🎉  
So you don't even need to run my code on your local computer since Codespaces allows you to edit in the browser now

I'm in the United States so it make take some time to make requested changes due to big time difference

"Allow edits from maintainers" is enabled ✔
- https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests


**My fork of XFlow: https://github.com/cliffordfajardo/XFlow**

CleanShot 2022-03-02 at 13 09 42@2x

File: README.md [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+    <XFlowCanvas config={useGrapConfig()}>
+      <CanvasScaleToolbar />
+      <CanvasMiniMap minimapOptions={{ width: 200, height: 120 }} />
+      <CanvasSnapline color="#1890ff" />
+    </XFlowCanvas>
+  </Xflow>
+)
+```
+
+### Step2 设置相关配置
```
Comment (lyn-boyu) [inline README.md:102]: this line is missing

File: README.md [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+    mousewheel: { enabled: true, zoomAtMousePosition: true },
+  })
+
+  /** Set the React node that the canvas needs to render, and the React content on the connection */
+  config.setNodeRender('NODE1', props => <Node1 {...props} />)
+  config.setEdgeRender('EDGE1', props => <Edge1 {...props} />)
+})
+```
+
+### Step3 画布渲染等逻辑操作
```
Comment (lyn-boyu) [inline README.md:125]: this line is missing

File: README.md [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+
+设置相关配置后, 我们就可以在 `onLoad` 方法里进行一些必要的业务逻辑操作，比如从服务端获取数据、执行布局算法、渲染画布内容、监听画布相关事件等。
```
Comment (lyn-boyu) [inline README.md:127]: this line is missing

File: README.md [hunk 4]
```diff
@@ -0,0 +1,259 @@
...
+<a href="https://qr.dingtalk.com/action/joingroup?code=v1,k1,rOHuvgq5s0EHDktyyQJffDE3ZAmHnbB2e6iwn/w4BKs=&_dt_no_comment=1&origin=11" target="_blank" rel="noopener noreferrer">
+  <img src="https://gw.alipayobjects.com/mdn/rms_19b204/afts/img/A*KuL6R7t0Xw0AAAAAAAAAAAAAARQnAQ" alt="X6/XFlow diagram visualization communication group 1" width="375" />
+</a>
+
+## How to Develop
+
+We use [pnpm](https://pnpm.io/workspaces) to manage the project, the directory structure is as follows:
+
+```
+├── packages
```
Comment (lyn-boyu) [inline README.md:205]: this line is missing

**Review by lyn-boyu — CHANGES_REQUESTED** [review]
Thanks for your contribution

**Comment by cliffordfajardo** [comment]
Ok will fix - thanks for the feedback

**Review by gujinyue1010 — COMMENTED** [review]
+1

**Pull request merged**